src/routes/libraries.spec.js

```diff
@@ -611,13 +582,13 @@ describe('/libraries', () => {
                 it('is an object with \'name\' and \'latest\' properties', () => {
                     for (const result of response.body.results) {
                         expect(result).to.have.property('name').that.is.a('string');
-                        expect(result).to.have.property('latest').that.is.a('string');
+                        try {
+                            expect(result).to.have.property('latest').that.is.a('string');
+                        } catch (_) {
+                            expect(result).to.have.property('latest').that.is.null;
+                        }
```

Comment:
Is this basically saying the property should either be a `string` or `null`?

Reply:
Indeed -- there are a few different ways to do this (calling typeof instead and then doing a oneOf against strings), but this is (IMO) the most "chai-esque" way to do this.